rpc/send_test.go

```diff
@@ -201,11 +200,12 @@ func TestClientNotReady(t *testing.T) {
 
 	// Construct a server that listens but doesn't do anything.
 	s := &Server{
-		Server:  rpc.NewServer(),
 		context: nodeContext,
 		addr:    addr,
+		methods: map[string]method{},
 	}
-	if err := s.Register(&Heartbeat{}); err != nil {
+	heartbeat := &Heartbeat{}
```

Comment:
Yeah, it's not really necessary. I pulled it out because `(&Heartbeat{}).Ping` is ugly. I re-inlined it, although it does need to be a reference.